### PR TITLE
Make kwargs explicit for Source component

### PR DESCRIPTION
**Pull request by srhbrnds**
This PR makes kwargs explicit for component Source.

### CODE REVIEW

File: src/oemof/solph/components/_source.py [hunk 1]
```diff
@@ -19,10 +19,42 @@
 
 
 class Source(on.Source):
-    """An object with one output flow."""
+    """A node object which is designed for one output flow.
```
Comment (jnnr) [inline src/oemof/solph/components/_source.py:22]:
```suggestion
    """A component which is designed for one output flow.
```

File: src/oemof/solph/components/_source.py [hunk 2]
```diff
@@ -19,10 +19,42 @@
...
+    >>> bel = solph.buses.Bus(label='electricity')
+
+    >>> pv_plant = solph.components.Source(
+    ...    label='pp_pv',
+    ...    outputs={bel: solph.flows.Flow()})
+
+    >>> type(pv_plant)
+    <class 'oemof.solph.components._source.Source'>
+
+    >>> str(pv_plant.outputs()])
```
Comment (joroeder) [inline src/oemof/solph/components/_source.py:44]:
```suggestion
    >>> str(pv_plant.outputs())
```

File: src/oemof/solph/components/_source.py (hunk 1 above)
Comment (jnnr) [reply src/oemof/solph/components/_source.py:22]: Just a suggestion to make it more specific

File: src/oemof/solph/components/_source.py [hunk 3]
```diff
@@ -19,10 +19,42 @@
...
+    It is theoretically possible to use the Source object with multiple outputs.
+    However, we strongly recommend using multiple Source objects instead.
+    """
+
+    def __init__(self, label=None, outputs=None):
+        if outputs is None:
+            outputs = {}
+
+        super().__init__(label=label, outputs=outputs)
         check_node_object_for_missing_attribute(self, "outputs")
```
Comment (srhbrnds) [inline src/oemof/solph/components/_source.py:58]: Function needs to be replaced by refactored function that raises error message and should be triggered in if-loop (line 55)

**Comment by srhbrnds** [comment]
- [ ] Run tox to check if tests are corrected @srhbrnds

**Pull request merged**